karton/core/config.py

```diff
@@ -39,10 +43,18 @@ def __init__(self, path: Optional[str] = None) -> None:
         self.config.read(self.SEARCH_PATHS)
         self._load_from_env()
 
-        if not self.config.has_section("minio"):
-            raise RuntimeError("Missing MinIO configuration")
-        if not self.config.has_section("redis"):
-            raise RuntimeError("Missing Redis configuration")
+        if check_sections:
+            if not self.config.has_section("minio"):
+                raise RuntimeError("Missing MinIO configuration")
+            if not self.config.has_section("redis"):
+                raise RuntimeError("Missing Redis configuration")
+
+    @staticmethod
+    def from_dict(config_dict):
```

Comment:
maybe?

```suggestion
    def from_dict(config_dict: Dict) -> Config:
```

(this will need additional import on the top)

Reply:
and also `"Config"` instead of `Config`.  Applied.